suitcase/tiff_stack/__init__.py

```diff
@@ -280,6 +284,14 @@ def event_page(self, doc):
                     tw = self._tiff_writers[streamname][field]
                     tw.save(img_asarray, *self._kwargs)
 
+    @staticmethod
+    def _get_prefixed_filename(file_prefix, start_doc, streamname, field):
+        '''Assemble the prefixed filename.'''
+        templated_file_prefix = file_prefix.format(
+            start=start_doc, field=field, streamname=streamname)
+        filename = f'{templated_file_prefix}{streamname}-{field}.tiff'
```

Comment:
Will it put `streamname` and `field` twice (first time in the `templated_file_prefix`, and the second one in the `filename` above)?

Reply:
Yes. The postfix must include them to ensure no collisions from within a given Run. The prefix may also include them (though it doesn’t do so by default) in which case it would occur twice. The use case that prompted this is putting them in separate directories, so it would occur in the _full path_ twice but just once in the file name.

Reply:
Related to (3) here https://github.com/bluesky/suitcase-core/issues/65#issue-435731549

Reply:
OK, thanks @danielballan, it makes sense now.